lib/rules/max-len.js

```diff
@@ -180,6 +188,59 @@ module.exports = {
         }
 
         /**
+         * Ensure that an array exists at [key] on `object`, and add `value` to it.
+         *
+         * @param {Object} object the object to mutate
+         * @param {string} key the object's key
+         * @param {*} value the value to add
+         * @returns {void}
+         * @private
+         */
+        function ensureArrayAndPush(object, key, value) {
+            if (!Array.isArray(object, key)) {
```

Comment:
Also, this suggests possible lack of test coverage in the "else case" here (there is no else, but I'm referring to this function being called twice in the same test case, where the second time should not reinitialize the array).